README.md

```diff
@@ -0,0 +1,52 @@
+# Proxless
+
+> Reduce your kubernetes cost by making all your deployments serverless with Proxless
+
+Proxless is an open-source proxy made for Kubernetes.
+Every deployment/service that are fronted by Proxless will be made serverless.
+Proxless scale to 0 your deployments after N seconds idle and scale them back to 1 after a call to the URL.
+
+## Installation
+
+A helm chart will be available soon.
```

Comment:
I think helm is good for complex services. I prefer the way people install directly from a URL:

kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml

Reply:
@Disturbing I can do both.
Reason being that I would want to enable or not ingress and autoscaling. Makes sense?